apps/public/src/app/(general)/samenvatting/Summary.tsx

```diff
@@ -17,16 +19,39 @@ type GenericSummaryData = {
 
 type Props = {
   additionalQuestions: GenericSummaryData[]
+  attachments: {
+    key: string
+    term: string
+    data: {
+      file: Blob
+      meta: {
+        originalFilename: string
+        contentType: string
+      }
+    }[]
+  }
   contact?: GenericSummaryData
   location: GenericSummaryData
   primaryForm: GenericSummaryData
 }
 
 const initialState: { message?: string } = {}
 
-export const Summary = ({ primaryForm, additionalQuestions, location, contact }: Props) => {
+export const Summary = ({ attachments, primaryForm, additionalQuestions, location, contact }: Props) => {
   const [formState, formAction] = useActionState(postSummaryForm, initialState)
 
+  const [fileList, setFileList] = useState<File[]>([])
+
+  useEffect(() => {
```

Comment:
Do we need this? `SummaryList` accepts a blob as well, right?

Reply:
Unfortunately, it does accept a Blob (except you'll get a type error), but it also misses the `name` attribute. So the images are shown but the title is missing. Couldn't find a way to add the name attribute to the Blob without it crashing.